tally_ho/apps/tally/views/tally_manager.py

```diff
@@ -381,3 +389,66 @@ def post(self, request, *args, **kwargs):
             'status': 'OK',
             'elements_processed': elements_processed}),
                             content_type='application/json')
+
+
+class SetUserTimeOutView(LoginRequiredMixin,
+                         mixins.GroupRequiredMixin,
+                         mixins.ReverseSuccessURLMixin,
+                         SuccessMessageMixin,
+                         UpdateView):
+    model = SiteInfo
+    form_class = SiteInfoForm
+    group_required = groups.TALLY_MANAGER
+    template_name = "tally_manager/set_user_timeout.html"
+    success_url = 'tally-manager'
+
+    def get_object(self):
+        site_id = self.kwargs.get('site_id', None)
+
+        return get_object_or_404(Site, pk=site_id)
+
+    def get(self, *args, **kwargs):
+        user_idle_timeout = None
+        self.object = self.get_object()
+
+        try:
+            siteinfo = SiteInfo.objects.get(site__pk=self.object.pk)
+            user_idle_timeout = siteinfo.user_idle_timeout
+        except SiteInfo.DoesNotExist:
+            user_idle_timeout = getattr(settings, 'IDLE_TIMEOUT', 60)
```

Comment:
can we move 60 to a `DEFAULT_IDLE_TIMEOUT` constant, defined in https://github.com/onaio/tally-ho/blob/master/tally_ho/settings/common.py probably makes sense

Reply:
Sure will do

Reply:
On second thought I think we should remove the hard coded constant completely and rename IDLE_TIMEOUT to DEFAULT_IDLE_TIMEOUT. This is because the current IDLE_TIMEOUT is already defined in common settings file, which is used in both development and production environments. Thoughts on this?.